homeassistant/components/plex/__init__.py

```diff
@@ -167,19 +167,30 @@ def update_plex():
     )
     hass.data[PLEX_DOMAIN][WEBSOCKETS][server_id] = websocket
 
-    async def async_start_websocket_session(_):
-        await websocket.listen()
+    async def async_start_websocket_session():
+        if hass.data[PLEX_DOMAIN][PLATFORMS_COMPLETED][server_id] == PLATFORMS:
+            hass.loop.create_task(websocket.listen())
```

Comment:
Why do we have to wait for the platforms?

Reply:
Once the websocket is created it can start updating devices, which may dispatch to create new `media_player` entities and update the `sensor`. If the dispatch listeners are not set up yet, those calls are lost and the entities are never created or the sensor fails to update.